modules/ui/fields/wikipedia.js

```diff
@@ -11,6 +11,8 @@ import { utilGetSetValue, utilNoAuto, utilRebind } from '../../util';
 
 
 export function uiFieldWikipedia(field, context) {
+  const scheme = 'https://';
+  const domain = 'wikipedia.org';
```

Comment:
This is reasonable, though FYI the _languageCode_.wikipedia.org subdomain pattern, `w/index.php` path, and `wiki/` rewrite rule are all specific to Wikipedia and [configurable by each wiki](https://www.mediawiki.org/wiki/Manual:Short_URL).

Reply:
the lang as subdomain is already being taken care by pre-existing logic. I believe.

Good catch about the specificities. Is this code meant to deal with non wikipedia rewrite rules? I would happily make it wiki specific, but it doesn't seem trivial to retrieve that from the result search metadata.